inspirehep/dojson/hepnames/rules.py

```diff
@@ -481,7 +487,7 @@ def advisors2marc(self, key, value):
         'a': value.get('name'),
         'g': value.get('degree_type'),
         'x': get_recid_from_ref(value.get('record')),
-        'y': '1' if value.get('curated_relation') else '0',
+        'y': '1' if value.get('curated_relation') else None,
```

Comment:
This is better than before but still maybe should not be there, since legacy does not know about this subfield.